pages/en/blog/announcements/v22-release-announce.md

```diff
@@ -0,0 +1,92 @@
+Maglev improves CPU performance for short-lived CLI programs.
+Contributed by Keyhan Vakil in [#51360](https://github.com/nodejs/node/pull/51360)
+
+#### Support require()ing synchronous ESM graphs
+
+This release adds `require()` support for synchronous ESM graphs under
+the flag `--experimental-require-module`.
+
+If `--experimental-require-module` is enabled, and the ECMAScript
+module being loaded by `require()` meets the following requirements:
+
+- Explicitly marked as an ES module with a "type": "module" field in the closest package.json or a .mjs extension.
+- Fully synchronous (contains no top-level await).
+
+`require()` will load the requested module as an ES Module, and return
+the module name space object. In this case it is similar to dynamic
+`import()` but is run synchronously and returns the name space object
+directly.
+
+Contributed by Joyee Cheung in [#51977](https://github.com/nodejs/node/pull/51977)
+
+#### Running package.json scripts
+
+Node.js 22 includes a new experimental feature for the execution of scripts from the `package.json` with the cli flag `node --run <script-in-package-json>`.
+It also supports `node run test` which executes the test command inside `package.json` scripts.
+Contributed by Yagiz Nizipli [#52190](https://github.com/nodejs/node/pull/52190)
+
+#### Stream default High Water Mark
+
+Increased the default High Water Mark for streams from 16KiB to 64KiB.
+This provides a performance boost across the board at the cost of slightly higher memory usage.
+Users in memory sensitive environments are encouraged to explicitly set `setDefaultHighWaterMark`.
+Contributed by Robert Nagy in [#52037](https://github.com/nodejs/node/pull/52037)
+
+#### Watch Mode (node --watch)
+
+From this release Watch Mode is considered stable.
+When in watch mode, changes in the watched files cause the Node.js process to restart.
+
+Contributed by Moshe Atlow [#52074](https://github.com/nodejs/node/pull/52074)
+
+#### Websocket
+
+The browser-compatible implementation of WebSocket, previously behind the flag `--experimental-websocket`, will be enabled by default.
+This provides a WebSocket client to Node.js without external dependencies.
+
+Contributed by the Undici team and Aras Abbasi in [#51594](https://github.com/nodejs/node/pull/51594)
+
+## Call to action
+
+Try out the new Node.js 22 release! We’re always happy to hear your feedback. Testing your applications and modules with Node.js 22 helps to ensure the future compatibility of your project with the latest Node.js changes and features.
+
+Also of note is that Node.js 18 will go End-of-Life in April 2025, so we advise you to start planning to upgrade to Node.js 20 (LTS) or Node.js 22 (soon to be LTS).
+
+Looking to the future, the [Next-10](https://github.com/nodejs/next-10) team is running a survey to gather info from the ecosystem. Help shape the future of Node.js by participating. Submit your feedback [here](TBD).
```

Comment:
```suggestion
Looking to the future, the [Next-10](https://github.com/nodejs/next-10) team is running a survey to gather info from the ecosystem. Help shape the future of Node.js by participating. Submit your feedback [here](TBD).
```
We still to defined the next-10 survery thats coming out next week

Reply:
We'll release it on tuesday, is there time for it?

Reply:
probably not so I'll remove it